common/utils/works.js

```diff
@@ -246,16 +239,54 @@ export function getItemAtLocation(work: Work, locationType: string): Item {
   return item;
 }
 
-export function getItemLocationsOfType(
+function itemIdentifierWithId(item: Item, id: string): boolean {
```

Comment:
Should these funcitons be named `isASpecialSnowFlake`?

Reply:
`renderWhatever({some, untyped, random, variables, that, leak, state})`